src/main/resources/config/liquibase/changelog/20240418092928_add_new_field.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet id="00000000000002" author="beyzaaltuntas">
+        <addColumn tableName="quiz_question">
+            <column name="content" type="varchar(5000)">
```

Comment:
```suggestion
            <column name="content" type="JSON">
```
Is it possible to use the JSON data type here? Both MySQL and PostgreSQL should support that.

The varchar solution will either silently truncate the string or fail the insert depending on MySQL settings. The JSON data type additionally ensures only valid JSON is stored.

Reply:
This PR only aims to see serialization of quiz questions into JSON object would work, in which type we store it in the database is a concern for future tasks.

Reply:
I do not agree. The database type should be done here already, as the entire point of this PR is to adapt the database schema.